cmd/bisync/log.go

```diff
@@ -29,16 +29,29 @@ func (b *bisyncRun) indent(tag, file, msg string) {
 		logf = fs.Logf
 	}
 
+	if b.opt.DryRun {
+		logf = fs.Logf
+		msg += " in dry-run mode"
+	}
+
 	if tag == "Path1" {
 		tag = Color(terminal.CyanFg, "Path1")
+	} else if tag == "Path2" {
+		tag = Color(terminal.CyanFg, "Path2")
 	} else {
 		tag = Color(terminal.BlueFg, tag)
 	}
+
 	msg = Color(terminal.MagentaFg, msg)
-	msg = strings.Replace(msg, "Queue copy to", Color(terminal.GreenFg, "Queue copy to"), -1)
-	msg = strings.Replace(msg, "Queue delete", Color(terminal.RedFg, "Queue delete"), -1)
+	msg = strings.Replace(msg, "Queue copy to", Color(terminal.GreenFg, "Queue copy to Path2"), -1)
```

Comment:
This part needs another look, as it could be either direction here (1 to 2 or 2 to 1). So, as it stands now it would result in confusing lines like:

```
NOTICE: - Path2    Queue copy to Path2 Path1 in dry-run mode - file.txt
```

Here's an example of how this function is called:
https://github.com/rclone/rclone/blob/6e28edeb9aa1c18e27faf641729b2cbc259fe13a/cmd/bisync/deltas.go#L479